### PR TITLE
Lk combine shard metrics

### PR DESCRIPTION
**Pull request by ekiernan**
### Description
Adding script for testing metrics

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: website/docs/Pipelines/Multiome_Pipeline/README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ The Multiome workflow calls two WARP subworkflows, one external subworkflow (opt
 | multimappers_Rescue_matrix | `UniqueAndMult-Rescue.mtx` | Optional output produced when `soloMultiMappers` is "Rescue"; see STARsolo [documentation](https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md#multi-gene-reads) for more information. |
 | multimappers_PropUnique_matrix | `UniqueAndMult-PropUnique.mtx` | Optional output produced when `soloMultiMappers` is "PropUnique"; see STARsolo [documentation](https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md#multi-gene-reads) for more information.|
 | gex_aligner_metrics | `<input_id>.star_metrics.tar` | Text file containing per barcode metrics (`CellReads.stats`) produced by the GEX pipeline STARsolo aligner. |
+| library_metrics | `<input_id>_library_metrics.csv` | CSV file with all library-level metrics calculated from STARsolo for gene expression data. |
```
Comment (kayleemathews) [inline website/docs/Pipelines/Multiome_Pipeline/README.md:123]:
```suggestion
| library_metrics | `<input_id>_library_metrics.csv` | Optional CSV file containing all library-level metrics calculated with STARsolo for gene expression data. |
```

File: website/docs/Pipelines/Optimus_Pipeline/README.md [hunk 2]
```diff
@@ -255,6 +255,7 @@ The following table lists the output files produced from the pipeline. For sampl
 | cell_metrics | `<input_id>.cell-metrics.csv.gz` | Matrix of metrics by cells. | Compressed CSV |
 | gene_metrics | `<input_id>.gene-metrics.csv.gz` |  Matrix of metrics by genes. | Compressed CSV |
 | aligner_metrics | `<input_id>.star_metrics.tar` | Tarred metrics files produced by the STARsolo aligner; contains align features, cell reads, summary, and UMI per cell metrics files. | TXT |
+| library_metrics | `<input_id>_library_metrics.csv` | CSV file with all library-level metrics calculated from STARsolo for gene expression data. | CSV |
```
Comment (kayleemathews) [inline website/docs/Pipelines/Optimus_Pipeline/README.md:258]:
```suggestion
| library_metrics | `<input_id>_library_metrics.csv` | Optional CSV file containing all library-level metrics calculated with STARsolo for gene expression data. | CSV |
```

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 3]
```diff
@@ -117,6 +117,7 @@ The Paired-Tag workflow calls two WARP subworkflows and an additional task which
 | gene_metrics_gex | `<input_id>_gex.gene_metrics.csv.gz` | CSV file containing the per-gene metrics. |
 | cell_calls_gex | `<input_id>_gex.emptyDrops` | TSV file containing the EmptyDrops results when the Optimus workflow is run in sc_rna mode. |
 | h5ad_output_file_gex | `<input_id>_gex.h5ad` | h5ad (Anndata) file containing the raw cell-by-gene count matrix, gene metrics, cell metrics, and global attributes. See the [Optimus Count Matrix Overview](../Optimus_Pipeline/Loom_schema.md) for more details. |
+| library_metrics | `<input_id>_library_metrics.csv` | CSV file with all library-level metrics calculated from STARsolo for gene expression data. |
```
Comment (kayleemathews) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:120]:
```suggestion
| library_metrics | `<input_id>_library_metrics.csv` | Optional CSV file containing all library-level metrics calculated with STARsolo for gene expression data. |
```

File: pipelines/skylab/slideseq/SlideSeq.changelog.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+# 3.1.1
```
Comment (kayleemathews) [inline pipelines/skylab/slideseq/SlideSeq.changelog.md:1]: @ekiernan Would you mind updating the Slide-seq pipeline version in the overview doc?

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md [hunk 5]
```diff
@@ -1,3 +1,8 @@
+# 1.3.1
```
Comment (kayleemathews) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md:1]: @ekiernan Would you mind updating the version in the Multi-snSS2 Overview and Methods docs as well?

**Review by kayleemathews — APPROVED** [review]
@ekiernan Thanks for updating docs! I made a couple of minor suggestions.

**Comment by ekiernan** [comment]
Running scientific Optimus tests here: #6​73​83

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ slug: /Pipelines/PairedTag_Pipeline/README
 
 | Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
 | :----: | :---: | :----: | :--------------: |
-| [PairedTag_v0.1.0](https://github.com/broadinstitute/warp/releases) | February, 2024 | Kaylee Mathews | Please file GitHub issues in warp or contact [documentation authors](mailto:warp-pipelines-help@broadinstitute.org) |
+| [PairedTag_v0.2.0](https://github.com/broadinstitute/warp/releases) | February, 2024 | Kaylee Mathews | Please file GitHub issues in warp or contact [documentation authors](mailto:warp-pipelines-help@broadinstitute.org) |
```
Comment (kayleemathews) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:10]:
```suggestion
| [PairedTag_v0.3.0](https://github.com/broadinstitute/warp/releases) | February, 2024 | Kaylee Mathews | Please file GitHub issues in warp or contact [documentation authors](mailto:warp-pipelines-help@broadinstitute.org) |
```

File: website/docs/Pipelines/SlideSeq_Pipeline/README.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ slug: /Pipelines/SlideSeq_Pipeline/README
 
 | Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
 | :----: | :---: | :----: | :--------------: |
-| [SlideSeq v3.1.0](https://github.com/broadinstitute/warp/releases) | February, 2024 | Elizabeth Kiernan & Kaylee Mathews | Please file GitHub issues in warp or contact [documentation authors](mailto:warp-pipelines-help@broadinstitute.org) |
+| [SlideSeq v3.1.1](https://github.com/broadinstitute/warp/releases) | February, 2024 | Elizabeth Kiernan & Kaylee Mathews | Please file GitHub issues in warp or contact [documentation authors](mailto:warp-pipelines-help@broadinstitute.org) |
```
Comment (kayleemathews) [inline website/docs/Pipelines/SlideSeq_Pipeline/README.md:10]:
```suggestion
| [SlideSeq v3.1.2](https://github.com/broadinstitute/warp/releases) | February, 2024 | Elizabeth Kiernan & Kaylee Mathews | Please file GitHub issues in warp or contact [documentation authors](mailto:warp-pipelines-help@broadinstitute.org) |
```

**Review by kevinpalis — APPROVED** [review]
Looks great.

**Review by khajoue2 — APPROVED** [review]

**Pull request merged**